eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Utilities/ResourceHelpers.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.TypeSpec.Generator.Input.Extensions;
 using Microsoft.TypeSpec.Generator.Primitives;
 using System;
+using System.Collections.Generic;
```

Comment:
why is this added?

Reply:
i wrote something then removed.
will clean this up